### PR TITLE
chore: split the creator and preview into multiple webpack configs

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

As part of the work to enable dynamic imports, this change splits the shell of the site and the preview into two applications that communicate through the iframe. This will allow separate logic for the preview such as adding the library scripts and requesting them to be dynamically imported from the shell UI.

## 👩‍💻 Reviewer Notes

The app should work exactly the same, this can be tested in FAST Creator using `yarn start`.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

### CODE REVIEW

File: sites/fast-creator/app/webpack.common.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const manifest = require("@microsoft/site-utilities/src/curated-html.json").join("");
+
+const appDir = path.resolve(__dirname, "./");
+const outDir = path.resolve(__dirname, "./www");
+
+module.exports = {
+    name: "root",
+    entry: {
+        main: path.resolve(appDir, "index.tsx"),
+        focusVisible: require.resolve(
```
Comment (nicholasrice) [inline sites/fast-creator/app/webpack.common.js:17]: :focus-visible shipped in chromium 86, so you may not need this anymore. Just FYI :)

**Review by nicholasrice — APPROVED** [review]

File: sites/fast-creator/app/webpack.common.js (hunk 1 above)
Comment (janechu) [reply sites/fast-creator/app/webpack.common.js]: Excellent! Thanks, that makes the bundle a wee bit smaller :)

**Review by chrisdholt — APPROVED** [review]

**Review by Ibrahimmaga — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
Sorry for the late feedback here. If we're splitting this into two parts, how hard would it be to have the preview app implementation be 100% web components?

**Comment by janechu** [comment]
@EisenbergEffect not hard, but I'd prefer to not add things that are not on the MVP list at this time. I'll create an issue in the the Creator repository for tracking.

**Pull request merged**